src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedParameterSymbol.cs

```diff
@@ -178,6 +178,11 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
             {
                 AddSynthesizedAttribute(ref attributes, moduleBuilder.SynthesizeIsReadOnlyAttribute(this));
             }
+
+            if (this.IsParams)
```

Comment:
>if (this.IsParams) [](http://example.com/codeflow?start=12&length=18)

Is there a guarantee that this type is always used to represent a parameter of a delegate Invoke method? Can we assert that? Otherwise, I think we should consider restricting the change to delegate Invoke methods only.

Reply:
Good catch. Yes, only delegates currently can have `IsParams: true`, I added asserts: 9129b221b063bed67f11ec4c0a7c71e49c979f65

Reply:
>>Is there a guarantee that this type is always used to represent a parameter of a delegate Invoke method? 

I assume the answer is no. Given that, the following suggestion still stands:

>>I think we should consider restricting the change to delegate Invoke methods only. 

Rather than just simply asserting inside the ```if```.

Reply:
Changed: 6133cf225ed29dd1bdce7ad32134033b8e6ac701